### ISSUE TITLE
Name column too short / hide columns feature

### ISSUE THREAD

**Issue by jonashaag** [bcicen/ctop#128]
When the terminal is too small, some columns are truncated.

<img width="1206" alt="bildschirmfoto 2018-05-07 um 08 09 31" src="https://user-images.githubusercontent.com/175722/39687304-0be69db6-51ce-11e8-84a5-0ddc32b831c9.png">
<img width="1199" alt="bildschirmfoto 2018-05-07 um 08 09 49" src="https://user-images.githubusercontent.com/175722/39687306-0d13cf56-51ce-11e8-90b4-f16a8cbc8b2f.png">

It's the name and CID column; CID is fine but name should be longer for my use case.

Ideas:

- Add hide column command line toggle. I'm rarely interested in the last 3 columns
- Be smarter about shrinking column width.
  - On small display, disable CPU and memory gauge and only display numbers. This decreases size of the CPU column in particular.

**Comment by demyxco** [comment]
I would also like this feature. Columns like CID and PIDs are not necessary for me on initial view.

**Comment by sir-gon** [comment]
Or CPU / Mem could be tighter.

**Comment by delanym** [comment]
Yes please

**Comment by andrejreznik** [comment]
Container names in docker compose have root dir prefix. This is just what is placed in the cells of the name column.

Would be nice if there is ellipsis in the middle of name instead of truncated name from the end.

**Comment by stokito** [comment]
You can disable CID column and this give you more space:
* Press `c` to open column settings
* Disable the `id` column by pressing `x`
* Exit from column settings by pressing `q` and save config by pressing `S` i.e. `Shift`+`s`

**Comment by stokito** [comment]
I don't think it worth to make some smart logic to hide columns. ctop already have a support of fixed size columns: PID and CID.
We can just set a container name to some fixed size that will satisfy almost all users. Images has a limit to 30 chars, I think this will be a reasonable width for container names too.
You may (almost) quickly disable CID or any other column to get more space.
This is how it looks in terminal with 80 chars width with disabled CID:
![ctop name col 30](https://user-images.githubusercontent.com/415502/100484700-2da0a680-3106-11eb-9cfb-62c787bb14bc.png)
